src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -291,6 +291,30 @@ const char* GetEnvValueBoolean(const char* envVariable)
     return (std::strcmp(envValue, "1") == 0 || strcasecmp(envValue, "true") == 0) ? "true" : "false";
 }
 
+static void *TryLoadHostPolicy(const char *hostPolicyPath)
+{
+#if defined(__APPLE__)
+    static const char LibrarySuffix[] = ".dylib";
+#else // Various Linux-related OS-es
+    static const char LibrarySuffix[] = ".so";
+#endif
+
+    size_t hostPolicyPathLength = sizeof(char) * strlen(hostPolicyPath);
```

Comment:
The rest of the file is on the `std::string` plan for string manipulation. It would look better for this to follow the suit.

Reply:
Fixed in 5th commit.